src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,104 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
Don't use Objects, Arrays, or any other util class. Instead of using Arrays.copyOf in the grow method, implement your own method to resize the array.